src/swap.rs

```diff
@@ -240,10 +241,8 @@ mod arbitrary {
                 hbit_params: hbit::Params::arbitrary(g),
                 herc20_params,
                 secret_hash: secret_hash(g),
-                utc_start_of_swap: chrono::NaiveDateTime::from_timestamp(
-                    u32::arbitrary(g) as i64,
-                    0,
-                ),
+                // todo:   chrono::NaiveDateTime::from_timestamp(u32::arbitrary(g) as i64, 0);
+                start_of_swap: chrono::Utc::now(),
```

Comment:
That is a bad idea as it will completely mess up the "[shrinking](https://github.com/BurntSushi/quickcheck#shrinking)" of quickcheck. You should be able to just get an `i64` from the `Gen` trait?